src/replay/service.zig

```diff
@@ -830,6 +832,18 @@ fn bypassConsensus(state: *ReplayState, results: []const ReplayResult) !bool {
         state.logger.info().logf("rooting slot with SlotTree.reRoot: {}", .{new_root});
         slot_tracker.root = new_root;
         slot_tracker.pruneNonRooted(state.allocator);
+        if (state.account_store == .accounts_db) {
+            // TODO: get this value from the right place
+            const lamports_per_signature = 5000;
```

Comment:
this is in slot constants

Reply:
7190fdc023c52cdd23b7a0d0cc0dda8ddc4279a9